plugins/woocommerce-admin/client/typings/index.d.ts

```
@@ -1,12 +1,16 @@
 declare module '@woocommerce/e2e-utils';
 declare module '@woocommerce/e2e-environment';
 declare module '@woocommerce/settings';
+declare module '@woocommerce/tracks';
 declare module '@wordpress/data';
 declare module '@wordpress/compose';
+declare module '@wordpress/plugins';
```

Hmm doing this just means that we will ignore any type definitions for everything imported from @wordpress/plugins

I think a better way is to install the typings from DefinitelyTyped : https://www.npmjs.com/package/@types/wordpress__plugins

And we should do it for the @wp/data, compose too.

For @woocommerce/tracks its one of our own packages and there aren't DefinitelyTyped definitions for it. I wonder if we should accept the error and create an issue to make type definitions for it?

---

Good catch! I didn't notice that they have `@types` packages.

> For @woocommerce/tracks its one of our own packages and there aren't DefinitelyTyped definitions for it. I wonder if we should accept the error and create an issue to make type definitions for it?

Yes, I think we could create an issue for this, but I'd like to ignore this error. Is that good?

---

Added @wp/data, compose, plugin type definitions in 8df6178d3b5126d9d6461aeacc0dd2dc054a54e2